Startup.cs

```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello World!w");
+                    await context.Response.WriteAsync("Hello World!");
```

Comment:
what is this bullshit, you can't just write hello world with only one exclamation mark

Reply:
We need to comply with our assumed test case in issues